best-practices/gke-batch-refarch/README.md

```diff
@@ -101,7 +101,7 @@ This reference architecture illustrates an example of a batch platform on GKE th
    a. Open a new terminal on your local machine and get the cluster credentials using the following command: 
 
    ```bash
-   gcloud container clusters get-credentials gke-batch-refarch --region $REGION --project $PROJECT_ID
+   gcloud container clusters get-credentials batch-dev --region $REGION --project $PROJECT_ID
```

Comment:
I suggest using a variable name for the cluster name

Reply:
I think the goal is to have people run it as is. We only added some substitutions to allow for multiple people to test in the same project.